packages/salesforcedx-vscode-apex/package.json

```diff
@@ -94,7 +94,7 @@
         "@salesforce/apex-tmlanguage": "1.4.0",
         "@salesforce/core": "3.31.18",
         "@salesforce/templates": "57.0.2",
-        "@salesforce/source-deploy-retrieve": "7.14.2",
+        "@salesforce/source-deploy-retrieve": "7.15.0",
```

Comment:
I bumped @salesforce/source-deploy-retrieve to pull in the latest, which includes the fast-xml-parser upgrade, but the Snyk build step is still failing.  I closed and opened the PR but that didn't do the trick.